pkg/api/rollupResolver.go

```diff
@@ -0,0 +1,24 @@
+package api
+
+//import ()
+
+// A single rollup
+
+type RollupResolver struct {
+	// TODO: add datestamp here? DATESTAMPS FOR EVERYONE
```

Comment:
Datestamps could be useful to pass to a graphing library on the frontend.  Each object will have the date to plot on the x axis and the data for the y axis.  Or perhaps any such library will be fine with a start/end date and the array indexes.  Revists when a we've determined the needs of the frontend library.

Reply:
Do this

Reply:
TODO: Rework rollups to not have null entries on dates with nod ata